<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QWidget" name="icon_only_widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="product_btn1">
           <property name="text">
            <string>P</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="offers_btn1">
           <property name="text">
            <string>O</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>520</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="exit_btn1">
         <property name="text">
          <string>E</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QWidget" name="full_menu_widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="product_btn2">
           <property name="text">
            <string>Product</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="offers_btn2">
           <property name="text">
            <string>Offers</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>520</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="exit_btn2">
         <property name="text">
          <string>Exit</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="sidebar_hide_btn">
            <property name="text">
             <string>Hide</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>493</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="user_btn">
            <property name="text">
             <string>User</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="autoExclusive">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="user_page">
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Username:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Total offers:</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="product_page">
          <widget class="QWidget" name="layoutWidget">
           <property name="geometry">
            <rect>
             <x>270</x>
             <y>300</y>
             <width>135</width>
             <height>51</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QLineEdit" name="pic_path"/>
            </item>
            <item>
             <widget class="QPushButton" name="upload_photo_btn">
              <property name="text">
               <string>Upload Photo</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="layoutWidget">
           <property name="geometry">
            <rect>
             <x>270</x>
             <y>390</y>
             <width>135</width>
             <height>149</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QLineEdit" name="name"/>
            </item>
            <item>
             <widget class="QLineEdit" name="price"/>
            </item>
            <item>
             <widget class="QLineEdit" name="description"/>
            </item>
            <item>
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="send_product_btn">
              <property name="text">
               <string>Send product</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>89</x>
             <y>10</y>
             <width>491</width>
             <height>271</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">image: url(default_photo.png);</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="textFormat">
            <enum>Qt::AutoText</enum>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <property name="openExternalLinks">
            <bool>false</bool>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="offers_page">
          <widget class="QTableWidget" name="offers_table_widget">
           <property name="geometry">
            <rect>
             <x>9</x>
             <y>9</y>
             <width>621</width>
             <height>321</height>
            </rect>
           </property>
           <attribute name="horizontalHeaderCascadingSectionResizes">
            <bool>true</bool>
           </attribute>
           <attribute name="horizontalHeaderMinimumSectionSize">
            <number>40</number>
           </attribute>
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>140</number>
           </attribute>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <column>
            <property name="text">
             <string>Название</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>ID</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Адрес доставки</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>ID заказа</string>
            </property>
           </column>
          </widget>
          <widget class="QPushButton" name="send_current_offer_btn">
           <property name="geometry">
            <rect>
             <x>520</x>
             <y>340</y>
             <width>111</width>
             <height>51</height>
            </rect>
           </property>
           <property name="text">
            <string>Send current offer</string>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>sidebar_hide_btn</sender>
   <signal>toggled(bool)</signal>
   <receiver>full_menu_widget</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>13</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sidebar_hide_btn</sender>
   <signal>toggled(bool)</signal>
   <receiver>icon_only_widget</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>208</x>
     <y>11</y>
    </hint>
    <hint type="destinationlabel">
     <x>52</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>product_btn1</sender>
   <signal>toggled(bool)</signal>
   <receiver>product_btn2</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>11</y>
    </hint>
    <hint type="destinationlabel">
     <x>102</x>
     <y>15</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>offers_btn1</sender>
   <signal>toggled(bool)</signal>
   <receiver>offers_btn2</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>37</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>108</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>product_btn2</sender>
   <signal>toggled(bool)</signal>
   <receiver>product_btn1</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>6</y>
    </hint>
    <hint type="destinationlabel">
     <x>65</x>
     <y>5</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>offers_btn2</sender>
   <signal>toggled(bool)</signal>
   <receiver>offers_btn1</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>104</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>62</x>
     <y>37</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
